Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/ConsentController.cs

```diff
@@ -0,0 +1,72 @@
+﻿using IdentityServer4.Services;
+using IdentityServer4.Stores;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Promact.Oauth.Server.Models.IdentityServer4;
+using Promact.Oauth.Server.Services;
+using System.Threading.Tasks;
+
+namespace Promact.Oauth.Server.Controllers
+{
+    /// <summary>
+    /// This controller processes the consent UI
+    /// </summary>
+    [ServiceFilter(typeof(SecurityHeadersAttribute))]
+    public class ConsentController : BaseController
+    {
+        private readonly Services.IConsentService _consent;
```

Comment:
name should be directly used like IConsentService instead of Service.IConsentService